lib/schema-controller.js

```diff
@@ -85,7 +85,7 @@ class SchemaController {
     if (isReady) {
       return
     }
-    this.validatorCompiler = this.compilersFactory.buildValidator(this.schemaBucket.getSchemas(), serverOption.ajv)
+    this.validatorCompiler = this.compilersFactory.buildValidator(this.schemaBucket.getSchemas('valid'), serverOption.ajv)
```

Comment:
For clarification, why `'valid'` here and `'ser'` below?

Reply:
Sorry, typo: I was tweaking if it could be useful to know if the `getSchemas` was called for the validator or for the serializer